docarray/array/mixins/io/pushpull.py

```diff
@@ -104,43 +127,55 @@ def gen():
             yield _tail
 
         res = requests.post(
-            f'{_get_cloud_api()}/v2/rpc/da.push', data=gen(), headers=headers
+            f'{_get_cloud_api()}/v2/rpc/artifact.upload', data=gen(), headers=headers
```

Comment:
I'm just wondering here why the API endpoint was renamed from `da.push` to `artifact.upload`
Isn't artifact related to finetuner ?

Reply:
Not specifically related to finetuner. Under the hood, every types of storage will use Hub's `artifact storage system`.
Doesn't matter docarray, finetuner model, or anything else in the future.